api/datadoghq/v2alpha1/datadogagent_types.go

```diff
@@ -1092,10 +1092,10 @@ type KubeletConfig struct {
 	// +optional
 	AgentCAPath string `json:"agentCAPath,omitempty"`
 
-	// PodResourcesSocket is the path to the pod resources socket, to be used to read pod resource assignments
-	// Default: `/var/lib/kubelet/pod-resources/kubelet.sock`
+	// PodResourcesSocketDir is the path to the folder where the pod resources socket is, to be used to read pod resource assignments
+	// Default: `/var/lib/kubelet/pod-resources/`
```

Comment:
```suggestion
	// PodResourcesSocketPath is the host path where the pod resources socket is stored
	// Default: `/var/lib/kubelet/pod-resources/`
```
Changed name (we have `DockerScoketPath` and `CriSocketPath` as precedents) and language to try staying consistent with other host paths that are mounted. Not sure how to word "easily" why it's used (the second part of your sentence I deleted).